docs/wallet_browser_extension/provider/error.md

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+  - wallet
+  - browser extension
+  - provider API
+---
+
+# Errors
+
+The Provider will reject any RPC request with an error object if the RPC request cannot be fulfilled.
+Every error object should have a `code` property that indicates the error type occurred.
+
+## Error Type
+
+```ts
+interface XPLLProviderRpcError extends Error {
+  code: number;
+  data?: unknown;
+}
+```
+
+!!! tip
+`XPLLProviderRpcError` is inherited from the native [`Error`][error] object.
```

Comment:
need indentaiton to make the content to be placed under the `tip`?

Reply:
solved by 86f2f28